lgc/patch/PatchResourceCollect.cpp

```diff
@@ -1711,8 +1711,6 @@ void PatchResourceCollect::matchGenericInOut() {
       auto &locMap = *locMapIt;
       if (m_shaderStage == ShaderStageFragment) {
         unsigned location = locMap.first;
-        if (m_pipelineState->getColorExportState().dualSourceBlendEnable && location == 1)
-          location = 0;
```

Comment:
This is not needed because `getColorExportFormat` will check `dualSourceBlendEnabled` and modify the location accordingly.